specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -696,9 +696,7 @@
         },
         "parameters": {
           "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
+          "additionalProperties": true,
```

Comment:
Could you share the back group for this change?

This is a breaking change of swagger.

Reply:
Reverted this change